packages/accounts-base/accounts-base.d.ts

```diff
@@ -204,26 +204,34 @@ export namespace Accounts {
     options?: { fields?: Mongo.FieldSpecifier | undefined }
   ): Promise<Meteor.User | null | undefined>;
 
+  interface SendEmailResult {
+    email: string;
+    user: any;
```

Comment:
we dont know the expected user?
instead of use any, we would to use Record<string, any> to ensure the user isnt passing a string, number, boolean....